apps/src/templates/instructions/ScrollButtons.jsx

```diff
@@ -75,12 +120,12 @@ const ScrollButtons = React.createClass({
       <div style={this.props.style}>
         <div
           ref="scrollUp"
-          onClick={this.props.onScrollUp}
+          onMouseDown={this.scrollStart.bind(this, true)}
```

Comment:
nit: true/false for up/down is probably okay, but a named parameter might be a bit more readable.  I had to go to the function to see what true/false meant.